packages/Webkul/Admin/src/Providers/AdminServiceProvider.php

```diff
@@ -17,6 +17,7 @@ public function boot()
         $this->loadRoutesFrom(__DIR__ . '/../Http/routes.php');
 
         $this->loadTranslationsFrom(__DIR__ . '/../Resources/lang', 'admin');
+        $this->publishes([__DIR__.'/../resources/lang' => resource_path('lang/vendor/webkul/admin')]);
```

Comment:
@ghermans Kindly change "resources" to "Resources"

Reply:
@vaishaliwebkul done.